manticore/core/worker.py

```diff
@@ -229,3 +230,24 @@ def start(self):
     def join(self):
         self._p.join()
         self._p = None
+
+
+class DaemonThread(WorkerThread):
+    """
+    Special case of WorkerThread that will exit whenever the main Manticore process exits.
+    """
+
+    def start(self, target: typing.Optional[typing.Callable] = None):
+        """
+        Function that starts the thread. Can take an optional callable to be invoked at the start, or can be subclassed,
+        in which case `target` should be None and the the `run` method will be invoked at the start.
+        :param target: an optional callable that will be invoked to start the thread. The callable should accept this
```

Comment:
I think a newline is required for docs
```suggestion

        :param target: an optional callable that will be invoked to start the thread. The callable should accept this
```